### PR TITLE
Add release notes for 3.4.36 and 3.5.2

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.5/release-3-5-2.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Summary of changes
+------------------
+
+This release contains:
+
+-   HTCondor-CE 4.0.1
+    -   4.0.0: [Major feature update](https://github.com/htcondor/htcondor-ce/releases/tag/v4.0.0)
+        -   SciTokens support
+        -   Disabled job retries
+        -   configuration reorganization
```
Comment (brianhlin) [inline docs/release/3.5/release-3-5-2.md]:
```suggestion
        -   Simplified remote CE requirements format for non-HTCondor batch systems
        -   Major configuration reorganization
```

File: docs/release/3.5/release-3-5-2.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    -   4.0.1: [bug fix](https://github.com/htcondor/htcondor-ce/releases/tag/v4.0.1)
+    -   [Some manual adjustments are requiread](/release/release_series#updating-to-htcondor-ce-4x)
```
Comment (brianhlin) [inline docs/release/3.5/release-3-5-2.md]: This is very important, I'd maybe add this to the `HTCondor-CE 4.0.1` main bullet above

Comment (timtheisen) [reply docs/release/3.5/release-3-5-2.md]: Making this the first bullet

File: docs/release/3.5/release-3-5-2.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+------------
+
+- OSG System Profiler verifies all installed packages, which may result in
+[excessively long run times](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3804).
+
+
+Containers
+----------
+
+Several containers are available and tagged as `stable` in accordance with our
```
Comment (brianhlin) [inline docs/release/3.5/release-3-5-2.md]:
```suggestion
Several containers are available and have been tagged as `stable` in accordance with our
```

**Pull request merged**